src/main/java/org/evgenysav/MechanicService.java

```diff
@@ -0,0 +1,137 @@
+package org.evgenysav;
+
+import org.evgenysav.util.FileActions;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.*;
+
+public class MechanicService implements Fixer {
+
+    private static final String[] DETAILS = new String[]{"Фильтр", "Фланец", "Втулка", "Вал", "Ось", "Свеча", "Масло", "ГРМ", "ШРУС"};
+    private static final String FILENAME = "orders";
+    private static final String FILENAME_PATH = "csv/" + FILENAME + ".csv";
+    private static final Random random = new Random();
+    private static final StringBuilder stringBuilder = new StringBuilder();
+
+    @Override
+    public Map<String, Integer> detectBreaking(Vehicle vehicle) {
+        int numberOfDetails = random.nextInt(DETAILS.length);
+        if (numberOfDetails == 0) {
+            return Collections.emptyMap();
+        }
+
+        Map<String, Integer> map = new HashMap<>();
+        Set<Integer> takenIndexes = new HashSet<>();
+        while (numberOfDetails != 0) {
+            int randomKey = random.nextInt(DETAILS.length);
+            if (takenIndexes.contains(randomKey)) {
+                while (takenIndexes.contains(randomKey)) {
+                    randomKey = random.nextInt(DETAILS.length);
+                }
+            }
+
+            takenIndexes.add(randomKey);
+            map.put(DETAILS[randomKey], random.nextInt(10) + 1);
+            numberOfDetails--;
+        }
+
+        printMapToFIle(vehicle, map);
+        return map;
+    }
+
+    @Override
+    public void repair(Vehicle vehicle) {
+        int vehicleId = vehicle.getVehicleId();
+        List<String> linesFromFile = null;
+        try {
+            linesFromFile = FileActions.getLinesFromFile(FILENAME);
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+
+        Iterator<String> iterator = linesFromFile.iterator();
+        while (iterator.hasNext()) {
+            String line = iterator.next();
+            String[] strings = getStringsFromLine(line);
+            if (strings[0].equals(String.valueOf(vehicleId))) {
+                iterator.remove();
+            }
+        }
+
+        try (FileWriter fileWriter = new FileWriter(FILENAME_PATH)) {
+            for (String s : linesFromFile) {
+                fileWriter.write(s + "\n");
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+    }
+
+    @Override
+    public int findVehicleIdByStringFromFile(String line) {
+        List<String> linesFromFile = null;
```

Comment:
I noticed that you create the same variable "linesFromFile" in two methods. I suggest that you move it to a global variable and rename it.